esmvalcore/_task.py

```diff
@@ -386,9 +386,15 @@ def _initialize_cmd(self):
         """Create an executable command from script."""
         diagnostics_root = DIAGNOSTICS.scripts
         script = self.script
-        script_file = (diagnostics_root / Path(script).expanduser()).absolute()
 
+        # Check if local diagnostic path exists
+        script_file = Path(script).expanduser().absolute()
         err_msg = f"Cannot execute script '{script}' ({script_file})"
+        if not script_file.is_file():
+            # Try diagnostics_root
+            script_file = (
+                diagnostics_root / Path(script).expanduser()
+            ).absolute()
```

Comment:
should we add to the error message that another path was tried? 
`err_msg += f" or ({script_file})"`

Reply:
I agree with Felicity here - please add a message saying the custom path didn't work, check it, running diag from base repo etc :beer: